Support/Testing/CircleCI/install-cmake.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+##
+## Copyright (c) 2014-present, Facebook, Inc.
+## All rights reserved.
+##
+## This source code is licensed under the University of Illinois/NCSA Open
+## Source License found in the LICENSE file in the root directory of this
+## source tree. An additional grant of patent rights can be found in the
+## PATENTS file in the same directory.
+##
+
+cmake_version="3.7"
+cmake_package_name="cmake-${cmake_version}.0-Linux-x86_64"
+cmake_archive="${cmake_package_name}.tar.gz"
+cd "/usr/local/"
```

Comment:
Don't `cd` to `/usr/local`, download the archive in `/tmp` and just unpack it to `/usr/local` by specifying `-C` to `tar`. That way, it'll install the files to `/usr/local`, but the (now useless) archive is going to be deleted when you save the docker image.

Reply:
Sounds good, updated to reflect that.